### DISCUSSION TITLE
Welcome to Oops-No-Internet Discussions!

### DISCUSSION THREAD

**Discussion by ImaginativeShohag** [ImaginativeShohag/Oops-No-Internet#11] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SashaProkipchuk** [comment]
Hi, your library is so cool, but I can't understand how to extend it using BaseNoInternetDialog. Could you share some example please?

**Reply by ImaginativeShohag** [reply]
Please check the "pendulum" and "signal" directories for 2 implementations from [here](https://github.com/ImaginativeShohag/Oops-No-Internet/tree/master/oopsnointernet/src/main/java/org/imaginativeworld/oopsnointernet/dialogs).

**Reply by SashaProkipchuk** [reply]
Thanks for reply and advice!) Good luck, man🤝
